mmtk/src/api.rs

```diff
@@ -469,7 +469,7 @@ pub extern "C" fn mmtk_pin_object(object: ObjectReference) -> bool {
     if mmtk_object_is_managed_by_mmtk(object.to_raw_address().as_usize()) {
```

Comment:
We may in the future replace this with a check for the immix space (bound check), which should be much cheaper.

Reply:
I might add a comment to the code so we remember that.